packages/textlint-tester/README.md

```diff
@@ -11,20 +11,91 @@
 1. Write tests by using `textlint-tester`
 2. Run tests by [Mocha](http://mochajs.org/ "Mocha")
 
+```sh
+$(npm bin)/mocha test/
+```
+
 ### TextLintTester
 
+`TextLintTester#run` has two signatures.
+
+- If you want to test single rule, use first method (`TextLintTester#run(ruleName, rule, {valid=[], invalid=[]})`)
+- If you want to test multiple rules and/or plugins, use second method (`TextLintTester#run(testName, testConfig, {valid=[], invalid=[]})`)
+
 #### TextLintTester#run(ruleName, rule, {valid=[], invalid=[]})
 
+- `{string} ruleName` ruleName is a name of the rule.
+- `{TextlintRuleCreator} rule` rule is textlint rule.
+
+#### TextLintTester#run(testName, testConfig, {valid=[], invalid=[]})
+
+- `{string} testName` testName is a name of the test.
+- `{TestConfig} testConfig` testConfig is the configuration object for the test.
+
+##### TestConfig object
+
+`TestConfig` is defined as follows:
+
+```typescript
+export declare type TestConfig = {
+    plugins: {
```

Comment:
`plugins?`

`plugins` is optional.

Reply:
@azu, thank you. Yes, `plugins` is optional. Reconsidering `isTestConfig` implementation...

```typescript
function isTestConfig(arg: any): arg is TestConfig {
    return arg.plugins !== undefined;
}
```